### PR TITLE
Publish release candidate

### PR DESCRIPTION
**Pull request by afallou**
**Why?**
To ease local development.

**What?**
Publish release candidate of the library for any branch but master.

### CODE REVIEW

**Review by dumpstate — APPROVED** [review]

**Review by JP-Globality — APPROVED** [review]

File: microcosm_flask/swagger/parameters/enum.py [hunk 1]
```diff
@@ -59,10 +59,11 @@ def parse_type(self, field: Field) -> str:
...
 
     def _parse_enum_values(self, field: Field) -> Sequence:
-        enum = getattr(field, "enum", None)
+        if not isinstance(field, EnumField):
+            raise Exception(f"Cannot parse enum values for non-enum fields: {field.name}")
         return [
-            choice.value if field.by_value else choice.name  # type: ignore
-            for choice in enum
+            choice.value if field.by_value else choice.name
+            for choice in field.enum
```
Comment (afallou) [inline microcosm_flask/swagger/parameters/enum.py:66]: Unrelated change to fix typehinting
